Shoko.Server/Providers/TraktTV/TraktTVHelper.cs

```diff
@@ -641,14 +641,27 @@ public void ScanForMatches()
                             }
                         }
 
-                        if (dictTraktSeasons != null && dictTraktSeasons.ContainsKey(xrefBase.TraktSeasonNumber))
+                        if (dictTraktSeasons != null && dictTraktSeasons.TryGetValue(xrefBase.TraktSeasonNumber, out var traktSeason))
                         {
-                            var episodeNumber = dictTraktSeasons[xrefBase.TraktSeasonNumber] +
-                                                (ep.EpisodeNumber + xrefBase.TraktStartEpisodeNumber - 2) -
-                                                (xrefBase.AniDBStartEpisodeNumber - 1);
-                            if (dictTraktEpisodes.ContainsKey(episodeNumber))
+                            int episodeNumber;
+                            {
```

Comment:
Why are you creating another scope just for the if/else statements?

Reply:
It did some weird things when I didn't do this before, going to re-check this and resolve

Reply:
looks like I had an error somewhere there before which is now gone. going to commit